plugins/TPChannelFilter.cpp

```diff
@@ -136,6 +140,17 @@ TPChannelFilter::do_work()
 
     // Actually do the removal for payload TPSets. Leave heartbeat TPSets unmolested
     if (tpset->type == TPSet::kPayload) {
+
+      if (m_first_tp) {
+        m_initial_offset = (duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count()) - (tpset->objects.begin()->time_start*(16*1e-6));
+        m_first_tp = false;
```

Comment:
Ditto with the `16*1e-6`. I wonder if it should be hard-coded somewhere in the configuration instead...

Reply:
as above, now moved to header(s)